### PR TITLE
Encrypter - Add ability to check if payload is encrypted

### PR DESCRIPTION
**Pull request by ericmp33**
Previous PR: https://github.com/laravel/framework/pull/53385

https://github.com/laravel/framework/pull/53385#issuecomment-2453574738 Sorry (,: added docblocks, tests & followed framework code style.

### CODE REVIEW

File: src/Illuminate/Encryption/Encrypter.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Encrypter implements EncrypterContract, StringEncrypter
      *
      * @var array
      */
-    private static $supportedCiphers = [
+    public static $supportedCiphers = [
```
Comment (crynobone) [inline src/Illuminate/Encryption/Encrypter.php:39]: Why do we need to change this?

**Review by crynobone — CHANGES_REQUESTED** [review]

File: src/Illuminate/Encryption/Encrypter.php (hunk 1 above)
Comment (ericmp33) [reply src/Illuminate/Encryption/Encrypter.php:39]: To be able to access it in the tests. If there is a better way to do it, please change it or let me know.

Comment (crynobone) [reply src/Illuminate/Encryption/Encrypter.php:39]: Should just manually use a data provider in tests, making this public may cause 3rd party to override the default and we really don't want that

Comment (ericmp33) [reply src/Illuminate/Encryption/Encrypter.php]: It's this better now? Or would you do it differently? @crynobone https://github.com/laravel/framework/pull/53407/commits/ffee580f012e14152156710efbb4a5421059620a

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If applicable, please consider releasing your code as a package so that the community can still take advantage of your contributions!

**Pull request closed without merging**